src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -109,24 +154,29 @@ Void deleteParticipantAndAllNeighbors(
     /**
      * Delete a study and their attached neighbors.  Returns the number of entities removed.
      *
-     * @param studyId - studyId
+     * @param organizationId - Id of the organization to which study belongs
+     * @param studyId        - studyId
      */
-    @DELETE( BASE + AUTHENTICATED + STUDY_ID_PATH )
+    @DELETE( BASE + AUTHENTICATED + ORGANIZATION_ID_PATH + STUDY_ID_PATH )
```

Comment:
Just want to make the note... this is not backwards compatible. I don't think this is in use yet, so I don't think it'll be a problem.

Reply:
Right!